pallets/parachain-staking/src/lib.rs

```diff
@@ -1315,19 +1332,30 @@ pub mod pallet {
 			});
 			<CandidatePool<T>>::put(candidates);
 		}
-		/// Compute round issuance based on total staked for the given round
-		fn compute_issuance(staked: BalanceOf<T>) -> BalanceOf<T> {
-			let config = <InflationConfig<T>>::get();
-			let round_issuance = crate::inflation::round_issuance_range::<T>(config.round);
-			// TODO: consider interpolation instead of bounded range
-			if staked < config.expect.min {
-				round_issuance.min
-			} else if staked > config.expect.max {
-				round_issuance.max
-			} else {
-				round_issuance.ideal
-			}
+
+		/// Compute total reward for round based on the amount in the reward pot
+		fn compute_total_reward_to_pay() -> BalanceOf<T> {
+			let total_unpaid_reward_amount = Self::reward_pot();
+			let mut payout = total_unpaid_reward_amount.checked_sub(&Self::locked_era_payout()).or_else(|| {
+				log::error!("💔 Error calculating era payout. Not enough funds in total_unpaid_reward_amount.");
+
+				//This is a bit strange but since we are dealing with money, log it.
+				Self::deposit_event(Event::NotEnoughFundsForEraPayment {reward_pot_balance: total_unpaid_reward_amount});
```

Comment:
Is this error used for anything else? If not, then it feels like we should also report the locked payout to help us find why that would be higher than the unpaid reward.

Reply:
We could, but if you see this event then you can query the balance of `locked_era_payout` at the time the event was raised. Having the `reward_pot_balance` and `locked_era_payout` on the event will not be enough to find the reason.